src/app/core/services/config/remote-config.service.ts

```diff
@@ -130,7 +130,17 @@ export class FirebaseRemoteConfigService extends RemoteConfigService {
     return this.fetch(this.timeoutMillis)
   }
 
+  protected sleep(milliseconds: number) {
+    const date = Date.now();
+    let currentDate = null;
+    do {
+      currentDate = Date.now();
```

Comment:
This is a busy-waiting sleep, really not recommended. Prefer to use actual timeouts.